omnibus/config/software/datadog-a7-py2.rb

```diff
@@ -1,26 +0,0 @@
-name "datadog-a7-py2"
-default_version "0.0.7"
-
-dependency "pip2"
-
-build do
-  ship_license "https://raw.githubusercontent.com/DataDog/datadog-checks-shared/master/LICENSE"
-
-  # aliases for the pips
-  if windows?
-    pip2 = "#{windows_safe_path(python_2_embedded)}\\Scripts\\pip.exe"
-    python2 = "#{windows_safe_path(python_2_embedded)}\\python.exe"
-  else
-    pip2 = "#{install_dir}/embedded/bin/pip2"
-    python2 = "#{install_dir}/embedded/bin/python2"
-  end
-
-  if windows?
-    # this pins a dependency of pylint->datadog-a7, later versions (up to v3.7.1) are broken.
-    command "#{python2} -m pip install configparser==3.5.0"
-    command "#{python2} -m pip install datadog-a7==#{version}"
-  else
-    command "#{pip2} install configparser==3.5.0"
-    command "#{pip2} install datadog-a7==#{version}"
```

Comment:
just realized by testing a nightly build that this also removes `pylint` altogether from the build. I'll open a PR to re-add `pylint` (and pin `configparser`)